Makefile

```diff
@@ -46,7 +46,13 @@ build-e2e:
 .PHONY: test-e2e
 test-e2e: ## Run openshift specific e2e test
 	go test -timeout 90m \
-		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e \
+		-v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e-operators \
+		-kubeconfig $${KUBECONFIG:-~/.kube/config} \
+		-machine-api-namespace $${NAMESPACE:-openshift-machine-api} \
+		-ginkgo.v \
+		-args -v 5 -logtostderr true
```

Comment:
why logtostderr?

Reply:
without the flag the logs get printed into a file instead of stderr

Reply:
We don't need the trailing `true`.

Reply:
Also why not same flag at L#58?

Reply:
This is a copied from what already exists.  In any case, I believe it's because that's how you capture the logs from pods.